packages/manager/src/features/CloudPulse/Alerts/CreateAlert/Criteria/DimensionFilterField.test.tsx

```diff
@@ -140,6 +143,16 @@ describe('Dimension filter field component', () => {
         },
       }
     );
+    const dataFieldContainer = container.getByTestId('data-field');
+    const dataFieldInput = within(dataFieldContainer).getByRole('button', {
+      name: 'Open',
+    });
+    await user.click(dataFieldInput);
+    await user.click(
+      await container.findByRole('option', {
+        name: dimensionFieldMockData[1].label,
+      })
+    );
```

Comment:
We shouldn't avoid nesting `await`

```suggestion
    const option = await container.findByRole('option', {
      name: dimensionFieldMockData[1].label,
    });
    await user.click(option);
```